packages/page-staking/src/Nomination/EraToTime.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { useApi} from '@polkadot/react-hooks';
+import { useTranslation} from '@polkadot/app-accounts/translate';
```

Comment:
Translation really needs to be pulled from this package and this package only. As it stands we are polluting the files, which makes translations a lot more work than it should be.